pkg/machines/libvirt-dbus.js

```diff
@@ -1045,9 +1057,22 @@ function startEventMonitorDomains(connectionName, dispatch) {
                     connectionName,
                     id: objPath,
                     updateOnly: true,
-                }));
-                // transient VMs don't have a separate Undefined event, so remove them on stop
-                dispatch(removeVm({ connectionName, id: objPath, transientOnly: true }));
+                }))
+                        .catch(ex => {
+                            if (!ex.message.includes("Domain not found"))
+                                return;
```

Comment:
Will other errors still be caught by something else and shown in the UI?

Reply:
No, we don't show in UI errors from the event handlers.